modules/data-loading/pages/data-streaming-connector/index.adoc

```diff
@@ -52,7 +61,12 @@ $ gadmin connector resume <connector_name>
 === Delete a connector
 Deleting a connector removes a connector.
 It stops the connector from streaming, as well as deletes the data that has been streamed to the corresponding topics in Kafka that haven't been ingested by a loading job.
+See the diagram in Figure 1 for details.
+
+
 This does not delete the topics themselves; they remain reusable since all data is deleted from the topics.
```

Comment:
This is confusing, can we word-smith it a bit. 

This does not delete the Kafka topics themselves, just the data in the topics associated to the connector.

Reply:
Updated